src/misc/paper-write.ts

```diff
@@ -0,0 +1,224 @@
+import { Material, Sound } from 'org.bukkit';
+import {
+  EquipmentSlot,
+  ItemStack,
+  PlayerInventory,
+} from 'org.bukkit.inventory';
+import { PlayerInteractEvent } from 'org.bukkit.event.player';
+import { Player } from 'org.bukkit.entity';
+import { color, text } from 'craftjs-plugin/chat';
+import { ChatColor } from 'net.md_5.bungee.api';
+import { isRightClick } from '../common/helpers/click';
+import { ChatMessage, GLOBAL_PIPELINE } from '../chat/pipeline';
+import { CustomItem } from '../common/items/CustomItem';
+import { VkItem } from '../common/items/VkItem';
+import * as yup from 'yup';
+import { isBuffer } from 'lodash';
+
+// Max length of text.
+const MAX_LENGTH = 128;
+
+const playersWriting: Player[] = [];
+
+/**
+ * CustomModelData table of Paper
+ * id 0: Normal paper
+ * 1: Written paper
+ * 2: Sealed paper
+ * 3: Empty envelope
+ * 4: Envelope with letter
+ * 5: Closed/sealed envelope
+ */
+
+export const envelopeWithLetter = new CustomItem({
```

Comment:
```suggestion
export const EnvelopeWithLetter = new CustomItem({
```
Capitalize the first letter of custom items